Start logging the time to index a project (#372)
atom/fuzzy-finder: docs/events.md

# Events specification

> This document specifies all the data (along with the format) which gets send from the Fuzzy Finder package to the GitHub analytics pipeline. This document follows the same format and nomenclature than the [Atom Core Events spec](https://github.com/atom/metrics/blob/master/docs/events.md).

Reviewer: ```suggestion
This document specifies all the data (along with the format) which gets sent from the Fuzzy Finder package to the GitHub analytics pipeline. This document follows the same format and nomenclature as the [Atom Core Events spec](https://github.com/atom/metrics/blob/master/docs/events.md).
```

## Counters

> Currently the Fuzzy finder does not log any counter event.

Reviewer: ```suggestion
Currently the Fuzzy finder does not log any counter events.
```

## Timing events

#### Time to crawl the project

* **eventType**: `fuzzy-finder-v1`
* **metadata**

  | field | value |
  |-------|-------|
  | `ec` | `time-to-crawl`
  | `el` | Crawler type (`ripgrep` or `fs`)
  | `ev` | Number of crawled files

## Standard events

> Currently the Fuzzy Finder does not log any standard event.

Reviewer: ```suggestion
Currently the Fuzzy Finder does not log any standard events.
```
